pkgs/applications/virtualization/docker/default.nix

```diff
@@ -195,6 +197,11 @@ rec {
       ./man/md2man-all.sh -q
 
       installManPage man/*/*.[1-9]
+    '' + ''
+      # systemd
+      mkdir -p $out/etc/systemd/system
+      ln -s ${moby}/etc/systemd/system/docker.service $out/etc/systemd/system/docker.service
+      ln -s ${moby}/etc/systemd/system/docker.socket $out/etc/systemd/system/docker.socket
```

Comment:
I think this needs to be moved under the ``optionalString (stdenv.isLinux)`` if.

Reply:
Ok.  Moved under optionalString(stdenv.isLinux).